### PR TITLE
Add infofeixe service to welcome-page

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Review by anacso17 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
   <meta content="width=device-width, initial-scale=1" name="viewport" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="description" content="Automation and Software welcome page.">
-  <meta name="author" content="Claudio Carneiro">
+  <!-- <meta name="author" content="Claudio Carneiro"> -->
```
Comment (fernandohds564) [inline index.html]: I think both should be kept as authors, don't you think?

**Review by RafaelLyra8 — APPROVED** [review]

File: template.mustache [hunk 2]
```diff
@@ -7,7 +7,8 @@
   <meta content="width=device-width, initial-scale=1" name="viewport" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="description" content="Automation and Software welcome page.">
```
Comment (anacso17) [inline template.mustache]: maybe change this description also to `Services for SIRIUS`

File: template.mustache [hunk 3]
```diff
@@ -7,7 +7,8 @@
...
-  <meta name="author" content="Claudio Carneiro">
+  <!-- <meta name="author" content="Claudio Carneiro"> -->
+  <meta name="author" content="Rafael Lyra">
   <link rel="icon" href="imgs/cropped-icon.png"/>
 
   <title>Automation and Software Group - Services</title>
```
Comment (anacso17) [inline template.mustache:13]: idem

File: index.html (hunk 1 above)
Comment (anacso17) [reply index.html]: Is this field "author" used in the GUI?

Comment (xresende) [reply index.html]: i think both should be kept as authors. this is why I did not delete the original line, just inserted a HTML comment. But although the line itself is not used in GUI (@anacso17), this meta data is indeed passed to the client interface. So maybe I should change the tag `author` to `authors` instead... This way, apart from git control version history, all authors should also go to web clients as meta data too.

Comment (xresende) [reply index.html]: done.

File: template.mustache (hunk 3 above)
Comment (xresende) [reply template.mustache:13]: done

**Pull request merged**